core/domain/exp_domain.py

```diff
@@ -459,7 +463,8 @@ def create_default_exploration(
             init_state_name=feconf.DEFAULT_INIT_STATE_NAME,
             category=feconf.DEFAULT_EXPLORATION_CATEGORY,
             objective=feconf.DEFAULT_EXPLORATION_OBJECTIVE,
-            language_code=constants.DEFAULT_LANGUAGE_CODE):
+            language_code=constants.DEFAULT_LANGUAGE_CODE,
+            image_id_counter=feconf.DEFAULT_IMAGE_ID_COUNTER):
```

Comment:
Do we have to pass `image_id_counter` while creating default exploration? (I think we should do it similar to what we have done below for version.)

Reply:
done